Workflow/Workflow_FailoverFailback.ps1

```diff
@@ -0,0 +1,232 @@
+<#
+The MIT License (MIT)
+
+Copyright © 2022 Tintri by DDN, Inc. All rights reserved.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+#>
+
+<#
+  This script demonstrates the failover and failback workflow.
+  
+  The code example/script provided here is for reference only to illustrate
+  sample workflows and may not be appropriate for use in actual operating
+  environments. 
+#> 
+Param(
+  [string] $source_tintriserver,
+  [string] $source_tsusername,
+  [string] $source_tspassword,
+  [string] $target_tintriserver,
+  [string] $target_tsusername,
+  [string] $target_tspassword,
+  [string] $viserver,
+  [string] $viusername,
+  [string] $vipassword,
+  [string] $vmname, # no FQDN
+  [string] $source_destination_directory, # no '/' at the end

```

Comment:
We have 2 places where we use this varaible, I guess it will be better to use it without datastore name in path. E.g., we want to keep new VM in datastore/dir_name, so we should point here only "dir_name"